src/libraries/Microsoft.PowerFx.Connectors/Execution/IConnectorInvoker.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors.Execution
+{
+    public interface IConnectorInvoker
```

Comment:
>IConnectorInvoker [](http://example.com/codeflow?start=21&length=17)

per offline discussion ... 

Consider making this interface internal or removing. 

If everybody actually just derives from FunctionInvoker, we can just use base class. 
And it sounds like users only override SendAsync, they don't override InvokeAsync, Context. So a user doesn't need this interface directly.